blog/src/Auth/Form/ResetForm.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Auth\Form;
+
+use App\User\UserRepository;
+use App\Auth\AuthService;
+use Yiisoft\Form\FormModel;
+use Yiisoft\Translator\TranslatorInterface;
+use Yiisoft\Validator\Result;
+use Yiisoft\Validator\Rule\Callback;
+use Yiisoft\Validator\Rule\Length;
+use Yiisoft\Validator\Rule\Required;
+use Yiisoft\Validator\ValidatorInterface;
+use Yiisoft\Validator\RulesProviderInterface;
+
+final class ResetForm extends FormModel implements RulesProviderInterface
+{
+    private string $login = '';
+    private string $password = '';
+    private string $password_verify = '';
```

Comment:
It is better to name properties as `passwordVerify`, not `password_verify`.

Reply:
I will make the needed change.